app/src/main/java/com/jerboa/ui/theme/Color.kt

```diff
@@ -1175,6 +1175,150 @@ fun woodland(): Pair<JerboaColorScheme, JerboaColorScheme> {
     return Pair(jerboaLight, jerboaDark)
 }
 
+fun dracula(): Pair<ColorScheme, ColorScheme> {
```

Comment:
Check the CI, this needs to match the above ones.

Reply:
yeah just copy pasted my code and missed that, hopefully it's fixed now